index.bs

```diff
@@ -529,6 +529,12 @@ dictionary MLOperandDescriptor {
 interface MLOperand {};
 </script>
 
+## MLOperator ## {#api-mloperator}
+<script type=idl>
+[SecureContext, Exposed=(Window, DedicatedWorker)]
+interface MLOperator {};
```

Comment:
For consistency, should we name it as `MLOperation`? This spec uses "operation" term in other places.

Reply:
Maybe it's just a wording thing but I tend to think of an *operation* as an *operator* with *operands* e.g. A + B is an operation, while the *operator* is just the act imposed on the operands e.g. + is the operator. 

I think of `conv2d` as an operation since it accepts operands as required params, while `relu` could either be an operation or an operator, depending on whether it is created with or without an operand. Does that make sense?

Reply:
Thanks for your explanation. It makes sense to me.